readme.md

```diff
@@ -34,20 +34,15 @@ Let **npm** do its thing. Once complete run:
 
 To make sure that the frontend built successfully, check the `frontend` directory for a new folder named `build` and note that it also updates the `node_modules` folder.
 
-### Add `invisible-flow-dump.sql` to your root directory
-The presence of the `invisible-flow-dump.sql` file in our project ensures that all contributors have the same schema and default data imported into their database when building the app with docker. 
+### Download CPDP Database
```

Comment:
This part of the readme could benefit from some rewording. The copy of CPDP doesn't need to be present in the project in order to run, and probably shouldn't be lest it be committed.